Tokenization_Samples/README.md

```diff
@@ -1,55 +1,81 @@
-                                                VTS samples
- 
- vts_tokenize
--------------------
+# VTS samples
+
+## tokenize
+
 Tokenize a given data or plaintext string using the tokengroup and tokentemplate.
 
-tokengroup : Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.
+**tokengroup** : `Defines a group name space in the configuration database. The VTS GUI Administrator must create token groups that match the tokengroup parameter names used in the application code.`
+
+**tokentemplate** : `VTS GUI Administrator-defined name for a group of properties that define tokenization operation.`
 
-tokentemplate : VTS GUI Administrator-defined name for a group of properties that define tokenization operation.
+**data** : `The data string to tokenize. The argument considers case and is limited to 128K.`
 
-data : The data string to tokenize. The argument considers case and is limited to 128K.
+**github link** : https://github.com/thalescpl-io/CipherTrust_Application_Protection/blob/TOK-2779/Tokenization_Samples/tokenize.py
 
-Syntax-
-$ python vts_tokenize.py -d -u <username>:<password> -l  <vts URL> -p <data or plaintext string> -O <outputFile> -tg <tokengroup> -tt <tokentemplate>
+**Syntax** :
 
-Example-
-$ python vts_tokenize.py -d -u root:ca\$hcow -l  https://192.0.2.0/vts -p DataToTokenizeDetokenize -O token -tg FF1_Tok_Group -tt FF1_Tok_Template
+```
+$ python tokenize.py -d -u <username>:<password> -l <vts URL> -p <data or plaintext string> -O <outputFile> -tg <tokengroup> -tt <tokentemplate>
+```
 
-Output : 
+**Example** :
 
-Header: {'User-Agent': 'python-requests/2.23.0', 'Accept-Encoding': 'gzip, deflate', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Type': 'application/json', 'Content-Length': '104', 'Authorization': 'Basic cm9vdDpjYSRoY293'}
+```
+$ python tokenize.py -d -u <username>:<password> -l https://192.0.2.0/vts -p DataToTokenizeDetokenize -O token -tg FF1_Tok_Group -tt FF1_Tok_Template
```

Comment:
Minor only:

How come the input is a URL? It should simply be an IP address or hostname of the Tokenization server. The scheme (https) and the "/vts" path of the URL should be hidden from the user and simply included in the URL of endpoint in your sample program altogether.

For example, the official REST-API endpoint of tokenize is this URL path:
/vts/rest/v2.0/tokenize